openwisp_monitoring/device/settings.py

```diff
@@ -1,38 +1,55 @@
 from django.conf import settings
 from django.core.exceptions import ImproperlyConfigured
 
+
+def get_or_create_critical_device_metrics():
+    # defines ``critical_device_metrics`` if they don't exist and validates them
+    critical_metrics = getattr(
+        settings,
+        'OPENWISP_MONITORING_CRITICAL_DEVICE_METRICS',
+        [{'key': 'ping', 'field_name': 'reachable'}],
+    )
+    for item in critical_metrics:  # pragma: no-cover
+        if not all(['key' in item, 'field_name' in item]):
+            raise ImproperlyConfigured(
+                'OPENWISP_MONITORING_CRITICAL_DEVICE_METRICS contains invalid items'
```

Comment:
"OPENWISP_MONITORING_CRITICAL_DEVICE_METRICS must contain the following keys: key, field_name" sounds better